docs/tutorial.md

```diff
@@ -30,68 +38,67 @@ Discourse requires connections to PostgreSQL and Redis. For more information, se
 
 Unit               Workload  Agent  Address      Ports  Message
-discourse-k8s/0*   active    idle   10.1.44.214         
-postgresql-k8s/0*  active    idle   10.1.44.219         
-redis-k8s/0*       active    idle   10.1.44.227         
+discourse-k8s/0*   active    idle   10.1.32.182
+postgresql-k8s/0*  active    idle   10.1.32.184         Primary
+redis-k8s/0*       active    idle   10.1.32.181
 
 Integration provider           Requirer                       Interface          Type     Message
-discourse-k8s:restart          discourse-k8s:restart          rolling_op         peer     
-postgresql-k8s:database        discourse-k8s:database         postgresql_client  regular  
-postgresql-k8s:database-peers  postgresql-k8s:database-peers  postgresql_peers   peer     
-postgresql-k8s:restart         postgresql-k8s:restart         rolling_op         peer     
-postgresql-k8s:upgrade         postgresql-k8s:upgrade         upgrade            peer     
-redis-k8s:redis                discourse-k8s:redis            redis              regular  
-redis-k8s:redis-peers          redis-k8s:redis-peers          redis-peers        peer     
+discourse-k8s:restart          discourse-k8s:restart          rolling_op         peer
+postgresql-k8s:database        discourse-k8s:database         postgresql_client  regular
+postgresql-k8s:database-peers  postgresql-k8s:database-peers  postgresql_peers   peer
+postgresql-k8s:restart         postgresql-k8s:restart         rolling_op         peer
+postgresql-k8s:upgrade         postgresql-k8s:upgrade         upgrade            peer
+redis-k8s:redis                discourse-k8s:redis            redis              regular
+redis-k8s:redis-peers          redis-k8s:redis-peers          redis-peers        peer
 ```
 The deployment finishes when all the charms show `Active` states.
 
-Run `kubectl get pods -n discourse` to see the pods that are being created by the charms:
-```
+Run `kubectl get pods -n discourse-tutorial` to see the pods that are being created by the charms:
+```bash
 NAME                             READY   STATUS    RESTARTS   AGE
-modeloperator-64c58d675d-csj47   1/1     Running   0          5m30s
+modeloperator-c584f6f9f-qf9gr    1/1     Running   0          5m30s
 redis-k8s-0                      3/3     Running   0          5m22s
 discourse-k8s-0                  2/2     Running   0          5m1s
 postgresql-k8s-0                 2/2     Running   0          5m9s
 ```
 
-### Provide ingress capabilities 
+### Provide ingress capabilities
 
 In order to expose the charm, the Nginx Ingress Integrator needs to be deployed and integrated with Discourse:
 
-```
+```bash
 juju deploy nginx-ingress-integrator
-# If your cluster has RBAC enabled you'll be prompted to run the following:
+#If your cluster has RBAC enabled, you'll be prompted to run the following (If you are working inside the Multipass vm, chances are you have RBAC enabled):
```

Comment:
I think that adding a way to check for RBAC wouldn't hurt here (`microk8s status`)